cea/optimization/preprocessing/decentralized_buildings.py

```diff
@@ -311,8 +312,9 @@ def calc_new_load(mdot, TsupDH, Tret, gv):
         BestData[building_name] = BestComb
 
     if 0:
-        os.chdir(locator.get_optimization_disconnected_folder())
-        fName = "DiscOpSummary.csv"
+        fName = locator.get_optimization_disconnected_folder_DiscOpSummary()
+        # os.chdir(locator.get_optimization_disconnected_folder())
```

Comment:
delete. BTW: thank you sooo much for getting rid of the `os.chdir`s! they have been driving me insane for waaay too long!